x-pack/plugins/threat_intelligence/public/components/empty_page/empty_page.tsx

```diff
@@ -0,0 +1,104 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import React, { VFC } from 'react';
+
+import { EuiEmptyPrompt, EuiImage, EuiButton, EuiLink } from '@elastic/eui';
+import { i18n } from '@kbn/i18n';
+import { FormattedMessage } from '@kbn/i18n-react';
+import { useTIDocumentationLink } from '../../hooks/use_documentation_link';
+import { useIntegrationsPageLink } from '../../hooks/use_integrations_page_link';
+import illustration from './integrations_light.svg';
+
+export const DOCS_LINK_TEST_ID = 'tiEmptyPageDocsLink';
+export const EMPTY_PROMPT_TEST_ID = 'tiEmptyPage';
+export const INTEGRATION_LINK_ID = 'tiEmptyPageIntegrationsPageLink';
+
+export const EmptyPage: VFC = () => {
+  const integrationsPageLink = useIntegrationsPageLink();
+  const documentationLink = useTIDocumentationLink();
```

Comment:
@PhilippeOberti think `useTIDcoumentationLink` should be `threatIntelInt` (check [here](https://github.com/elastic/kibana/blob/main/packages/kbn-doc-links/src/get_doc_links.ts#L342) for reference). I'll let @lcawl confirm though.

Reply:
It seems to be declared in x-pack/plugins/threat_intelligence/public/hooks/use_documentation_link.tsx, so LGTM